dev/src/main/java/org/eclipse/codewind/intellij/core/cli/CLIUtil.java

```diff
@@ -85,19 +93,20 @@ public static Process runCWCTL(String[] globalOptions, String[] cmd, String[] op
 
         List<String> cmdList = new ArrayList<String>();
         cmdList.add(codewindInfo.getInstallPath());
-		addOptions(cmdList, globalOptions);
-		addOptions(cmdList, cmd);
-		addOptions(cmdList, options);
-		addOptions(cmdList, args);
+        addOptions(cmdList, globalOptions);
+        addOptions(cmdList, cmd);
+        addOptions(cmdList, options);
+        addOptions(cmdList, args);
+        Logger.log(cmdList.stream().collect(Collectors.joining(" ")));
         String[] command = cmdList.toArray(new String[cmdList.size()]);
         ProcessBuilder builder = new ProcessBuilder(command);
         if (PlatformUtil.getOS() == PlatformUtil.OperatingSystem.MAC) {
-            String pathVar = System.getenv("PATH");
-            pathVar = "/usr/local/bin:" + pathVar;
+            String pathVar = System.getenv(CoreConstants.PATH);
```

Comment:
Hey John, on Windows, would this pathVar value have Windows specific paths?  Then on the next line, `/usr/local/bin` will be prepended to it.

Reply:
Hey John, never mind, I think this looks ok.